sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefServiceTest.java

```diff
@@ -14,20 +14,16 @@
 import org.lfenergy.compas.scl2007b4.model.*;
 import org.lfenergy.compas.sct.commons.dto.*;
 import org.lfenergy.compas.sct.commons.exception.ScdException;
+import org.lfenergy.compas.sct.commons.model.epf.*;
+import org.lfenergy.compas.sct.commons.model.epf.TChannel;
```

Comment:
Remove unused import epf.TChannel (since you imported epf.*)

Reply:
Done